packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -2832,12 +2831,13 @@ export const audiusBackend = ({
     if (isreadSubscribersFromDiscoveryEnabled) {
       // Read subscribers from discovery
       try {
-        const subscribers: APIUser[] = await audiusLibs.User.getUserSubscribers(
-          encodeHashId(userId)
-        )
-        const subscriberIds = subscribers.map((subscriber) =>
-          decodeHashId(subscriber.id)
-        )
+        const encodedUserId = encodeHashId(userId)
+        const bulkResp: { user_id: string; subscriber_ids: string[] }[] =
+          await audiusLibs.User.bulkGetUserSubscribers([encodedUserId])
+        const encodedSubscriberIds = bulkResp.find(
+          (user) => user.user_id === encodedUserId
+        )!.subscriber_ids
```

Comment:
Ideally we don't use !, but if it's always the user in the first spot of the array, can maybe just destructure instead of using find?